scripts/tasks/webpack.js

```diff
@@ -1,4 +1,18 @@
 // @ts-check
 
+const { argv } = require('just-task');
 const { webpackTask } = require('just-task-preset');
+const path = require('path');
+const fs = require('fs');
+
 exports.webpack = webpackTask();
+exports.webpackDevServer = function() {
+  const webpackConfigFilePath = argv().webpackConfig || 'webpack.serve.config.js';
+  const configPath = path.resolve(process.cwd(), webpackConfigFilePath);
+
+  if (fs.existsSync(configPath)) {
+    const webpackDevServerPath = path.resolve(__dirname, '../node_modules/webpack-dev-server/bin/webpack-dev-server.js');
+    const execSync = require('../exec-sync');
+    execSync(`node ${webpackDevServerPath} --config ${configPath} --open`);
+  }
```

Comment:
This (and other exec commands) could run into trouble if the repo is inside a folder with spaces in the name (especially on Windows). Probably something to fix all-up in a separate PR.

Reply:
This is why I have a different exec function inside `just-task-preset` that has an encoding capability for this reason. I hadn't exposed that API yet, but I could and then we can get rid of these execSync's in a later PR.

Reply:
That would be good. Also I'm curious what method you used for encoding and whether it will work in all scenarios...as described [here](https://github.com/Microsoft/web-build-tools/pull/1003) and in the linked Node issue, getting exec encoding right is complicated.